### PR TITLE
replace sheet to zeego dropdown in RSelect

### PR DESCRIPTION
**Pull request by Isthisanmol**

### CODE REVIEW

File: packages/ui/src/RSelect/index.tsx [hunk 1]
```diff
@@ -64,83 +54,62 @@ export function SelectItem(props) {
...
-  }
-
   return (
     <Select
       value={value}
       disablePreventBodyScroll
       onValueChange={handleChange}
       {...forwardedProps}
     >
-      <Select.Trigger width={220} iconAfter={ChevronDown}>
```
Comment (andrew-bierman) [inline packages/ui/src/RSelect/index.tsx:97]: Nice job but I think we should support both via prop is my thinking

just select with prop of native = sheet
Select with prop native + zeego = zeego dropdown

Comment (Isthisanmol) [reply packages/ui/src/RSelect/index.tsx:97]: @andrew-bierman Could you please review the code? If it is correct, I will apply it to the other instances where we use dropdown menus.

Comment (andrew-bierman) [reply packages/ui/src/RSelect/index.tsx:97]: yes this looks good, nice job :)

**Comment by JewelTee** [comment]
@Isthisanmol the dropdown looks great! Can we also change the dropdown from the weight unit when adding new item? So it won't occupy most of the screen. Thank You! Please see image below: 
Screenshot_20240621_143056

**Comment by JewelTee** [comment]
@Isthisanmol everything looks good. Thank You!

@andrew-bierman - this is ready for merging 👍🏻

File: packages/app/components/Dropdown.tsx [hunk 2]
```diff
@@ -116,34 +116,45 @@
```
Comment (andrew-bierman) [inline packages/app/components/Dropdown.tsx:116]: we should probs move this into ui package, not sure why its in components (not your fault, just noticed this)

**Review by andrew-bierman — APPROVED** [review]

File: packages/app/components/Dropdown.tsx (hunk 2 above)
Comment (Isthisanmol) [reply packages/app/components/Dropdown.tsx:116]: Done that.

**Pull request merged**